.gitignore

```diff
@@ -5,3 +5,4 @@ docs
 node_modules
 .eslintcache
 tsconfig.tsbuildinfo
+componentsjs-error-state.json
```

Comment:
Out of curiosity: what does this file do?

Reply:
Still have to write docs on this, but this file will be generated if dep injection crashes. It contains some state information that makes debugging a lot easier.